source/api/presentation/3.0/index.md

```diff
@@ -164,11 +173,11 @@ The value of the property _MUST_ be a JSON object, as described in the [language
  * A Canvas _SHOULD_ have the `label` property with at least one entry.<br/>
    Clients _MUST_ render `label` on a Canvas, and _MUST_ generate a `label` for Canvases that do not have them.
```

Comment:
This is at odds with line 1185 (5.3 Canvas), which says

> Every Canvas _SHOULD_ have a `label` to display. If one is not provided, the client _SHOULD_ automatically generate one for use based on the Canvas's position within the `items` property.

Reply:
I think _SHOULD_ for generate, it seems too ill-defined for _MUST_

Reply:
:+1: to SHOULD generate.